src/test/java/io/knotx/commons/http/request/AllowedHeadersFilterTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
```

Comment:
Let's keep that 2019, that's the year, module was introduced.
```suggestion
 * Copyright (C) 2019 Knot.x Project
```

Reply:
Done